ion_functions/data/adcp_functions.py

```diff
@@ -717,47 +746,54 @@ def vadcp_beam_vertical_true(b1, b2, b3, b4, b5, h, p, r, vf):
                     (b) moved the conditioning of compass readings to adcp_inst2earth.
         2015-06-22: Russell Desiderio. Renamed this data product.
         2015-06-25: Russell Desiderio. Incorporated b5 int fillvalue -> Nan.
+        2019-08-13: Christopher Wingard. Adds functionality to compute a 3-beam solution
+                    and cleans up syntax used in the function.
 
     Usage:
 
-        ww_true = vadcp_beam_vertical_true(b1, b2, b3, b4, b5, h, p, r, vf)
+        w = vadcp_beam_vertical_true(b1, b2, b3, b4, b5, pg1, pg2, pg3, pg4, pg5, h, p, r, vf)
 
             where
 
-        ww_true = true vertical velocity profiles in Earth coordinates
-                  (VELTURB-VLU-5BM_L1) [m s-1]
-
-        b1 = "beam 1" velocity profiles in beam coordinates (VELTURB-B1_L0) [mm s-1]
-        b2 = "beam 2" velocity profiles in beam coordinates (VELTURB-B2_L0) [mm s-1]
-        b3 = "beam 3" velocity profiles in beam coordinates (VELTURB-B3_L0) [mm s-1]
-        b4 = "beam 4" velocity profiles in beam coordinates (VELTURB-B4_L0) [mm s-1]
-        b5 = "beam 5" velocity profiles in beam coordinates (VELTURB-B5_L0) [mm s-1]
+        w = true vertical velocity profiles in Earth coordinates (VELTURB-VLU-5BM_L1) [m s-1]
+
+        b1 = beam 1 velocity profiles in beam coordinates (VELTURB-B1_L0) [mm s-1]
+        b2 = beam 2 velocity profiles in beam coordinates (VELTURB-B2_L0) [mm s-1]
+        b3 = beam 3 velocity profiles in beam coordinates (VELTURB-B3_L0) [mm s-1]
+        b4 = beam 4 velocity profiles in beam coordinates (VELTURB-B4_L0) [mm s-1]
+        b5 = beam 5 velocity profiles in beam coordinates (VELTURB-B5_L0) [mm s-1]
+        pg1 = percent good estimate for beam 1 [percent]
+        pg2 = percent good estimate for beam 2 [percent]
+        pg3 = percent good estimate for beam 3 [percent]
+        pg4 = percent good estimate for beam 4 [percent]
+        pg5 = percent good estimate for beam 4 [percent]
```

Comment:
Make pg5 be the percent good estimate for beam 5, not 4 (i.e. copy paste error duplicated the pg4 description).

Reply:
Done